### PR TITLE
V2/new overview page

### PR DESCRIPTION
**Pull request by dospore**
Update overview page to new designs https://www.figma.com/file/gA9jQ5B6ZwWjGKBhGUFbKo/2.0-3.0-Pools?node-id=1911%3A170292. This branch needs a bit of work in responsive styling but there are no designs for it so hard to tell what to do for it so think its ok to merge in as is until we get an update on that since this branch differs a lot to current release-v2

For now am ignoring the graph.

- merge commit history onto overview page
- merge pending commits onto overview page
- remove overview page routing

### CODE REVIEW

File: archetypes/Portfolio/OverviewTable/styles.tsx [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+export const TitleContent = styled.div`
+    display: flex;
+    gap: 0.75rem;
+`;
+
+export const Title = styled.div`
+    line-height: 150%;
+    font-weight: 600;
+    font-size: 20px;
```
Comment (dweberdev) [inline archetypes/Portfolio/OverviewTable/styles.tsx:33]: Update font-sizes to theme.fontSize throughout this file

File: archetypes/Portfolio/QueuedCommits/styles.tsx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import styled from 'styled-components';
+
+export const TimeLeftWrapper = styled.div`
+    font-size: 24px;
```
Comment (dweberdev) [inline archetypes/Portfolio/QueuedCommits/styles.tsx:4]: Update to theme.fontSize

File: archetypes/Portfolio/OverviewTable/styles.tsx (hunk 1 above)
Comment (dospore) [reply archetypes/Portfolio/OverviewTable/styles.tsx]: done

**Review by dweberdev — APPROVED** [review]
lgtm

**Pull request merged**